contrib/src/test/scala/akka/stream/contrib/PartitionWithSpec.scala

```diff
@@ -94,4 +97,23 @@ trait PartitionWithSpec extends BaseStreamSpec {
       sub2.expectNext(3, 5, 7, 9, 11, 13, 15, 17, 19, 21)
     }
   }
+
+  "partitionWith extension method" should {
+    "be callable on Flow and partition its output" in {
+      import PartitionWith.Implicits._
+
+      val flow = mergeFanOut(
```

Comment:
I wonder if we could zip the fanned-out elements instead of merging them? Right now the test just peforms a `map` in a roundabout way ;).

Reply:
Done.